src/App.jsx

```diff
@@ -1,33 +1,12 @@
 import { useState } from 'react'
-import reactLogo from './assets/react.svg'
+import Calculator from './Components/Calculator'
 import './App.css'
 
 function App() {
```

Comment:
It will be great to make this component a class-based component. Just like it has been mentioned in the previous review. It is a requirement to make this component class-based.